pkg/http/utils.go

```diff
@@ -324,8 +323,8 @@ func GetRequestBuffer(
 
 func GenericJSONPostClient(url string, json string) (*http.Response, error) {
 	data := []byte(json)
-	client := &http.Client{Timeout: time.Second * 1}
-	req, err := http.NewRequest("POST", url, bytes.NewBuffer(data))
+	client := newRetryClient()
+	req, err := retryablehttp.NewRequest("POST", url, bytes.NewBuffer(data))
```

Comment:
why didn't we use the `client` that was initialize at 326?

Reply:
some node  report timeout here, so add a retry here